contracts/FleekERC721.sol

```diff
@@ -41,8 +46,21 @@ contract FleekERC721 is ERC721, FleekAccessControl {
         string gitRepository;
     }
 
-    Counters.Counter private _tokenIds;
+    /**
+     * The stored data for each mirror.
+     * The mirror "score" is used to determine the best mirror to use.
+     * The mirror "score" equals to 0 means that the mirror is not verified.
+     */
+    struct Mirror {
+        uint256 tokenId;
+        uint256 index;
+        uint256 score;
+        address owner;
```

Comment:
Should we add domain here? Also should we add a type to see if its DNS mirror or Fleek zone type

Reply:
The domain would be the mapping key identifier.